packages/i3.rb

```diff
@@ -40,6 +27,19 @@ def self.install
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
     Dir.chdir "#{CREW_DEST_PREFIX}/bin" do
       system "for f in \$(ls #{CREW_BUILD}-*); do g=\$(echo \$f | sed 's,#{CREW_BUILD}-,,'); ln -sf \$f \$g; done"
+     system "touch /usr/local/bin/starti3"
+     system "echo '#!/bin/sh' >> /usr/local/bin/starti3"
+     system "echo 'stopsommelier' >> /usr/local/bin/starti3"
+     system "echo 'export DISPLAY=100.115.92.2:0' >> /usr/local/bin/starti3"
+     system "echo 'i3 & $1 $2 $3 $4' >> /usr/local/bin/starti3"
+     system "chmod +x /usr/local/bin/starti3"
     end
   end
+  
+  def self.postinstall
+      puts
+      puts "To use this package, you need to download XServer XSDL from Google Play Store".lightblue
+      puts "Use 'starti3' instead of 'i3' to execute this package".lightblue
+      puts "Open XServer XSDL before running 'starti3'"
```

Comment:
Might want to mention instructions on how to restore to using sommelier.  ie. `export DISPLAY=:0`, for example.

Reply:
Added to the script directly

Reply:
Have you tested this and run spacefm immediately afterwards to see if it launches?

Reply:
I have tested it with `starti3 firefox` and it works well but haven't test it with spacefm yet

Reply:
You can test with any gui app really.  I just used spacefm as an example.